CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2.7.0
+
+- Support has been implemented to change the character encoding for the connection, this makes it possible to change the default encoding from utf8 to win1252, iso8859, among others.
+
+- implemented executing a prepared statement with question mark placeholder style similar to PHP PDO
```

Comment:
Let's separate the two parts into two PRs: one for the encoding, the other for the `?` placeholder.

Reply:
I don't know if I'll have time to separate this into 2 PR
I think it would be kind of complicated to make this separation now
because I didn't do it in separate branches